### ISSUE TITLE
Please add additional columns

### ISSUE THREAD

**Issue by kittylyst** [callForPapers/callForPapers.github.io#40]
Love the site - really useful resource. A few suggestions for additional columns that I'd like to see:

CoC - does the conf have a Code of Conduct or an explicit diversity policy?
Commercial - Is this a commercial or community conf?
Transparent Finances - does the conf publish its finances?
Compensates Speakers - does the conf compensate its speakers? If so, to what level - travel & accom or speaking fees as well?

As a speaker, knowing these details matters. I am more than prepared to speak at a small community conf that's getting started & can't afford to pay my expenses. A commercial conf that presents itself as a community event whilst not paying speakers & making a large profit - not so much.

**Comment by cmatskas** [comment]
Hi @kittylyst, like you, I see a lot of value in knowing this information. However, I'm afraid that the current layout/format is not the best as it will (most likely) add horizontal scrolling and hide columns by default if we add more stuff.

I'm really open to suggestions on how to better present the data to the users. The site has a lot of potential. Please be aware that all the information we present is through manual effort and we would love to see conferences contributing to the site directly (IMHO)

Thanks for your suggestions

**Comment by kkokosa** [comment]
Yes, it would be REALLY nice to have those additional columns. But besides layout, there is another problem - I believe, very often those information are not clear and directly stated (by conferences public websites). Hence, the ideal way would be to promote site among conferences organizers, as @cmatskas mentioned...

**Comment by cmatskas** [comment]
Some conferences and sources I use provide some information. Maybe we could start by leaving it blank where we have no information and add it as we go. If the site get's better traction, we could ask for organisers to kindly provide us with that information. The site needs a bit of beautification to be honest.

Maybe we can get a designer to give us a hand :)

**Comment by kkokosa** [comment]
I would rally like to propose some re-design - but still not have time for that. Please give me a few days more:)

**Comment by cmatskas** [comment]
Still early days so plenty of scope to redesign stuff. Take your time :)

**Comment by kkokosa** [comment]
@cmatskas shall we open new issue for the redesign discussion? What do you think?

**Comment by cmatskas** [comment]
Yes. I think that this one should be about the additional columns. The redesign should be totally separate and maybe reference existing issues as well (if any) around design concerns

**Comment by kkokosa** [comment]
Great, so let's move redesign discussions to #42 :)
